packages/@aws-cdk/aws-scheduler-alpha/test/integ.schedule.ts

```diff
@@ -47,64 +57,57 @@ namedGroup.applyRemovalPolicy(cdk.RemovalPolicy.DESTROY);
 const unnamedGroup = new scheduler.Group(stack, 'UnnamedGroup', {});
 unnamedGroup.applyRemovalPolicy(cdk.RemovalPolicy.DESTROY);
 
-new scheduler.Schedule(stack, 'DefaultSchedule', {
-  schedule: expression,
-  target: target,
-});
-
-new scheduler.Schedule(stack, 'NamedGroupSchedule', {
-  schedule: expression,
-  target: target,
+const randomTargetThatWontGetInvoked = new SomeSqsTarget(queue, role);
```

Comment:
until April 2060 😆

Reply:
are the tests with this target just to check that the different schedule configurations deploy? Since I don't see any assertions I'm wondering if unit tests already cover these cases or if we should add assertions to make these tests more rigorous.

Reply:
These tests are from the old code. But my understanding is same as yours, i.e. these tests verifies the generated template deploys without failures. This can verify, for example, the dates are in correct format I assume.